tests/create_task_streamable.yml

```diff
@@ -0,0 +1,48 @@
+name: Create Tes Task Job With Input Streamable Flag
+description: Job to create a new TES Task with streamable flag set in input
+service: TES
+compatible_versions:
+  - v1.1
+tags:
+  - logical
+  - create task
+  - create task test
+  - all
+jobs:
+  - name: create_task
+    description: Create a new TES task
+    endpoint: /tasks
+    operation: POST
+    request_body: |
+      {
+        "name": "CompTest",
+        "description": "CompTest",
+        "executors": [
+          {
+            "image": "ubuntu:20.04",
+            "command": [
+              "TestString1",
+              "TestString2"
+            ]
+          },
+          {
+            "image": "ubuntu:20.04",
+            "command": [
+              "TestString3",
+              "TestString4"
+            ]
+          }
+        ],
+        "input": [
+          {
+            "name": "access-file",
+            "description": "Access file resource via streaming",
+            "url": "s3://my-object-store/file1",
+            "path": "/data/file1",
+            "streamable": true
```

Comment:
The `optional_filetype` and `streamable` tests are just schema checks specifically for TES `v1.1`. Can we combine both tests? It will check both schemas in a single task.

Reply:
I think there is value in keeping them separate. It's more explicit, and if the behavior of one of these changes in the future, it will be clear where the problem is.

Reply:
Sure, for better visibility we can keep them separate. But if there is any change in behavior, we will need to create another test for that newer version as it wont be supported by v1.1.0.

Reply:
And I think that's totally fine. _Every_ test case is tied only to a specific set of versions (see comment above).

But assuming we have figured out a way how to run the right tests for a given version, running a test run and having separate tests for `streamable` and the optional file type is convenient for the user: they will not have to try to disentangle whether a failed test is because of the one or the other or both.